package.json

```diff
@@ -39,6 +39,7 @@
 		"ink-big-text": "^2.0.0",
 		"ink-gradient": "^3.0.0",
 		"is-git-repository": "^2.0.0",
+		"magicc": "^0.2.1",
```

Comment:
Why did you install the package? @Abhinav-Bansal751, for doing the test, kindly read our contributing file. There's a guide there on how to test it locally.

Reply:
Did you remove this? @Abhinav-Bansal751 if yes click "resolve conversation"